src/app/zap-templates/zcl/data-model/chip/scene.xml

```diff
@@ -0,0 +1,235 @@
+<?xml version="1.0"?>
+<!--
+Copyright (c) 2022 Project CHIP Authors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+<configurator>
+  <bitmap name="ScenesCopyMode" type="BITMAP8">
+    <cluster code="0x0005"/>
+    <field name="CopyAllScenes" mask="0x01"/>
+  </bitmap>
+
+  <struct name="AttributeValuePair">
+    <cluster code="0x0005"/>
+    <item name="attributeId" type="attrib_id" optional="true"/>
+    <item name="attributeValue" type="INT8U" array="true"/>
```

Comment:
This is probably OK for now as we try to stand this up, but it doesn't really match the spec.  What we probably _really_ want here is a new type definition which gets mapped in the underlying C++ to a TLVReader positioned on the relevant element or some such....

Reply:
Right, I kinda knew it wasn't exactly what the spec was mentioning. But didn't find anything that could support variables types. I will ping our zap team to help me create such a type. For now, I added a TODO comment.